lib/ObisNames.js

```diff
@@ -363,89 +363,95 @@ const obisMeasurement = {
-            99: 'Cooling',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List'
+            1: "Cooling",
+            99: "Cooling",
+            96: "Service entry",
+            97: "Error message",
+            98: "List"
         },
         'de': {
-        	1: 'Kälte',
-        	99: 'Kälte',
-            96: 'Wartungseintrag',
-            97: 'Fehler',
-            98: 'Listen'
+            1: "Kälte",
+            99: "Kälte",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen"
         },
         "nl": {
-            1: "koud",
-            99: 'Cold',
+            1: "Koeling",
+            99: "Koeling",
             96: "Onderhoudsgegevens",
-            97: "Fout",
-            98: "Lijsten"
+            97: "Foutmelding",
+            98: "Lijst"
         }
     },
     6: { // Heating
         'en': {
-            1:  'Heat',
-            3:  'Heat',
-            99: 'Heat',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List'
+            1:  "Heat",
+            3:  "Heat",
+            99: "Heat",
+            96: "Service entry",
+            97: "Error message",
+            98: "List"
         },
         'de': {
-            1: 'Wärme',
-            3: 'Wärme',
-        	99: 'Wärme'
+            1: "Wärme",
+            3: "Wärme",
+            99: "Wärme"
+	    96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen"
         },
         "nl": {
-            1: "warmte",
-            3: 'Heat',
-            99: "Heat"
+            1: "Warmte",
+            3: "Warmte",
+            99: "Warmte"
+	    96: "Onderhoudsgegevens",
+            97: "Foutmelding",
+            98: "Lijst"
         }
     },
     7: { // Gas
         'en': {
-            128: 'Gas',
-            129: 'Gas',
-            96: 'Service entry',
-            97: 'Error message',
-            98: 'List',
-            99: 'Data profiles, Load profiles P.01/ P.02, Operation Log P.98/ P.99'
+            128: "Gas",
+            129: "Gas",
+            96: "Service entry",
+            97: "Error message",
+            98: "List",
+            99: "Data profiles, Load profiles P.01/ P.02, Operation Log P.98/ P.99"
         },
         'de': {
-            128: 'Betriebskubikmeter',
-            129: 'Normkubikmeter',
-            96: 'Wartungseintrag',
-            97: 'Fehler',
-            98: 'Listen',
-            99: 'Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99'
+            128: "Betriebskubikmeter",
+            129: "Normkubikmeter",
+            96: "Wartungseintrag",
+            97: "Fehler",
+            98: "Listen",
+            99: "Datenprofile, Lastgang P.01/ P.02, Betriebslogbuch P.98/ P.99"
         },
         "nl": {
-            128: "operationele kubieke meters",
-            129: "Standaard kubieke meter",
+            128: "Bedrijfskuub",
+            129: "Normaalkuub",
```

Comment:
ist dit niet gewoon "Gasverbruikl" ?

Reply:
Ik weet niet wat achter deze waarden zit. Ik heb een vertaling gezocht die het dichtst bij de Engelse en Duits vertaling zit. 

Ik heb dit via Google gevonden:

Onder normaalkuub wordt verstaan het aantal kubieke meters bij een temperatuur van 273 K en een luchtdruk van 101,3 kPa. Of te wel bij 0 graden Celsius en atmosferisch

Onder bedrijfskuub zal dan worden verstaan, bij de geldende druk en temperatuur van het proces.